### PR TITLE
master: sync up with zeus-dev

### PR DESCRIPTION
**Pull request by diego-sueiro**
IGN 2019-11-11 d168e60 Switch to zeus branch
2019-11-21 33e8d10 Add layers to poky for psa storage support (#135)

IGN(ored) = Changes that are specific to zeus-dev or already cherry picked

### CODE REVIEW

File: default.xml [hunk 1]
```diff
@@ -8,20 +8,20 @@
   <remote fetch="http://git.yoctoproject.org" name="yocto"/>
   <remote fetch="https://source.codeaurora.org/external" name="CAF"/>
 
-  <default revision="master" sync-j="4"/>
+  <default revision="zeus" sync-j="4"/>
```
Comment (jonathanhaigh-arm) [inline default.xml]: This doesn't look right to me.

Comment (diego-sueiro) [reply default.xml]: Good catch.
I accidentally added the d168e60 commit. Just removed and force-pushed.

**Comment by diego-sueiro** [comment]
Commit cf4117e added to unpin meta-freescale.

**Comment by diego-sueiro** [comment]
Added commit 57f14b9 to pin meta-linaro since the upstream layer updated the optee-* to 3.7.0 and we are using the 3.6.0.

**Comment by diego-sueiro** [comment]
Dropped the test commit.

File: poky.xml [hunk 2]
```diff
@@ -3,13 +3,16 @@
...
-  <default revision="warrior" sync-j="4"/>
+  <default revision="master" sync-j="4"/>
 
-  <project name="armmbed/meta-mbl" path="layers/meta-mbl" remote="github" revision="warrior-dev"/>
+  <project name="armmbed/meta-mbl" path="layers/meta-mbl" remote="github"/>
   <project name="git/meta-freescale" path="layers/meta-freescale" remote="yocto"/>
+  <project name="git/meta-security" path="layers/meta-security" remote="yocto"/>
   <project name="git/poky" path="layers/poky" remote="yocto"/>
+  <project name="openembedded/meta-openembedded" path="layers/meta-openembedded" remote="github"/>
+  <project name="openembedded/openembedded-core" path="layers/openembedded-core" remote="github"/>
```
Comment (jonathanhaigh-arm) [inline poky.xml]: Doesn't the poky repo (that we're putting in layers/poky) already have a copy of oe-core?

Comment (diego-sueiro) [reply poky.xml]: Yes, it does. I'll raise a ticket to address this duplicity.

Comment (jonathanhaigh-arm) [reply poky.xml]: I don't really like leaving it like this, but it's not really a problem with the zeus-dev->master merge, it's a problem with the code in zeus-dev that's being merged, so, okay.

Comment (jonathanhaigh-arm) [reply poky.xml]: (Will approve when there's a reference to the new ticket).

Comment (diego-sueiro) [reply poky.xml]: https://jira.arm.com/browse/IOTMBL-2675

**Pull request merged**